src/world/area_tik/tik_05/8750D0.c

```diff
@@ -10,6 +10,26 @@
 
 #include "world/common/CheckItemFlags40.inc.c"
 
+#include "battle/battle.h"
+#include "effects.h"
+
+#ifndef NON_MATCHING
+
+void play_model_animation(s32, s32);                     /* extern */
+extern s32 D_000001E4;
+
+ApiStatus func_8024093C_8756FC(Evt* script, s32 isInitialCall) {
+    Entity* entity = get_entity_by_index(evt_get_variable(script, *script->ptrReadPos));
```

Comment:
api funcs should define an "args" temp (see other script functions) and use that (*args++) rather than use script->ptrReadPos directly